packages/xo-web/src/common/intl/messages.js

```diff
@@ -1156,13 +1156,15 @@ const messages = {
   vifMacAutoGenerate: 'Auto-generated if empty',
   vifAllowedIps: 'Allowed IPs',
   vifNoIps: 'No IPs',
-  vifDisabledNetwork: 'Network disabled',
-  vifLockedNetwork: 'Network locked',
+  vifLockingModeDisabled: 'VIF locking mode is disabled',
+  vifLockingModeUnlocked: 'VIF locking mode is unlocked',
+  vifLockingModeLocked: 'VIF locking mode is locked',
+  networkDefaultLockingModeDisabled: 'Network default locking mode is disabled',
+  networkDefaultLockingModeUnlocked: 'Network default locking mode is unlocked',
   vifLockedNetworkNoIps:
     'Network locked and no IPs are allowed for this interface',
   vifUnlockedNetworkWithIps:
```

Comment:
There is an edge case on this [conditions](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/xo-app/vm/tab-network.js#L195) which is if the `lockingMode` is set to `network_default` and the network is enabled/disabled. 

Should the vif locking mode set to `locked` has the same behavior as vif locking mode set to `network_default` with a locked network?